rust/osm2lanes/src/lib.rs

```diff
@@ -119,7 +119,7 @@ impl LanePrintable for Lane {
                 designated: LaneDesignated::Bus,
                 ..
             } => '🚌',
-            Self::Shoulder => 'ˢ',
+            Self::Shoulder => '🛆',
```

Comment:
Hmm, my browser and terminal both have trouble with this. Might just be an issue with my system

Reply:
I'm fine with this one, I'll see if I can install that font...